collections/_article/dev-snapshot-godot-4-4-dev-1.md

```diff
@@ -0,0 +1,169 @@
+
+### Optimizations
+Contributors have been hard at work on various optimizations across the engine. Now that Godot 4 is being used more and
+the core feature set is feeling stable, we can turn our minds to optimizing the existing features. Many optimizations
+were finished and tested before the 4.3 release but were too late to merge. So now we get them all in one big batch.
+Here are a few:
+- [Core] Return memory allocations in when formatting strings ([GH-94558](https://github.com/godotengine/godot/pull/94558)).
+    This makes formatting strings about 2X faster.
+- [Core] Optimize Array.insert ([GH-94353](https://github.com/godotengine/godot/pull/94353)). This makes inserting new
+    elements into an Array about 10X faster on average. 
+- [Core] Optimize `String.insert` ([GH-92555](https://github.com/godotengine/godot/pull/92555)).
+- [Core] Optimize `String.join` ([GH-92550](https://github.com/godotengine/godot/pull/92550)).
+- [Core] Optimize `String.replace` ([GH-92546](https://github.com/godotengine/godot/pull/92546)).
+- [Import] Disable normal raycaster by default ([GH-93727](https://github.com/godotengine/godot/pull/93727)). This makes
+    importing meshes with LODs approximately 2X faster. We expect that the quality of imported LODs will be the same.
+    If you notice any issues, please let us know. You can re-enable the raycaster in the object's import settings under
+    `lods/raycast_normals`.
+- [Editor] Optimize editor grid recalculation ([GH-92734](https://github.com/godotengine/godot/pull/92734)).
+- [Editor] Optimize closing the editor settings ([GH-95704](https://github.com/godotengine/godot/pull/95704)).
+- [Rendering] Optimize meshes for vertex cache ([GH-94241](https://github.com/godotengine/godot/pull/94241)). In scenes
+    with a lot of geometry this can result in a 20% improvement in frame time. 
+- [Rendering] Optimize materials with `ambient_light_disabled` ([GH-92213](https://github.com/godotengine/godot/pull/92213)).
+
+
+### And more!
+
+There are too many exciting changes to list them all here, but here are a few more. 
+
+- [Editor] Fix `uid://` paths fail to load at editor startup ([GH-95689](https://github.com/godotengine/godot/pull/95689)).
+- [Editor] Correct rotation gizmo plane math for off-center perspective view ([GH-95402](https://github.com/godotengine/godot/pull/95402)).
+- [Editor] Make viewport grid visible on all three planes in ortho camera view ([GH-93869](https://github.com/godotengine/godot/pull/93869)).
+- [Editor] Expose GDScript syntax highlighter to editor plugins ([GH-95764](https://github.com/godotengine/godot/pull/95764)).
+- [Rendering] Fix generated light probes placing too close to manual light probes ([GH-83497](https://github.com/godotengine/godot/pull/83497)).
+- [Rendering] Antialias direct light samples in LightmapperRD ([GH-95828](https://github.com/godotengine/godot/pull/95828)).
```

Comment:
I feel this one is worth mentioning in the bicubic sampling paragraph as well, given how they tend to work together to provide the best results.

Reply:
Added.